### PR TITLE
Minor cleanup to evolve NewtonianEuler

### PR DESCRIPTION
**Pull request by carmaza**
## Proposed changes

Extra upgrades needed to evolve the NewtonianEuler system. Executable and files related will come in another PR.

### Types of changes:

- [ ] Bugfix
- [ ] New feature

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

File: src/Evolution/Systems/NewtonianEuler/Characteristics.hpp [hunk 1]
```diff
@@ -70,6 +70,8 @@ struct CharacteristicSpeedsCompute : CharacteristicSpeeds<Dim>, db::ComputeTag {
       tmpl::list<Velocity<DataVector, Dim>, SoundSpeed<DataVector>,
                  ::Tags::Normalized<::Tags::UnnormalizedFaceNormal<Dim>>>;
 
+  using volume_tags = tmpl::list<>;
```
Comment (kidder) [inline src/Evolution/Systems/NewtonianEuler/Characteristics.hpp]: this shouldn't be necessary as it is the default.

**Review by kidder — CHANGES_REQUESTED** [review]
group review with @nilsdeppe and @wthrowe

File: src/Evolution/Systems/NewtonianEuler/Characteristics.hpp (hunk 1 above)
Comment (carmaza) [reply src/Evolution/Systems/NewtonianEuler/Characteristics.hpp]: Removed!

**Comment by nilsdeppe** [comment]
IWYU is unhappy

**Comment by carmaza** [comment]
Same issue as in #1347, where we decided to ignore it.

**Review by nilsdeppe — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Comment by kidder** [comment]
Ignoring IWYU as it wants the same lines to be both removed and added...

**Pull request merged**